tasks/main.yml

```diff
@@ -40,6 +40,31 @@
     - /usr/bin/sonar
     - /etc/init.d/sonar
 
+- name: ensure sonar RUN_AS_USER is set correctly
+  lineinfile:
+    dest: /usr/local/sonar/bin/linux-x86-64/sonar.sh
+    regexp: ^#RUN_AS_USER
+    line: "RUN_AS_USER=sonar"
```

Comment:
In case someone wants to change the user name, I would suggest:

    regexp: ^#?RUN_AS_USER
    line: "RUN_AS_USER={{ sonar_service_account }}"